helm/jsonapi/templates/stargate_docsapi_deployment.yaml

```diff
@@ -30,7 +30,7 @@ spec:
       - name: jsonapi
         image: "{{ .Values.image.registry }}/{{ .Values.image.repository }}:{{ .Values.image.tag }}"
         ports:
```

Comment:
Not changed in this PR but isn't YAML file name misleading wrt `docsapi` part?

Reply:
make sense, renamed the file names.